kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kms/vault/Vault.java

```diff
@@ -102,7 +103,10 @@ private String getIngressDomain(boolean openshiftCluster) {
      */
     public void delete() throws IOException {
         LOGGER.info("Deleting Vault in {} namespace", deploymentNamespace);
-        NamespaceUtils.deleteNamespaceWithWait(deploymentNamespace);
+        String testSuiteName = ResourceManager.getTestContext().getRequiredTestClass().getName();
+        String testCaseName = ResourceManager.getTestContext().getTestMethod().orElse(null) == null ? ""
+                : ResourceManager.getTestContext().getRequiredTestMethod().getName();
```

Comment:
You would normally mix a ternary option and functional style in the way you've done here.

I *think* this is functionally the same (untested).
```suggestion
        String testCaseName = ResourceManager.getTestContext().getTestMethod().orElse("");
```